docs/how-to-guides/create-backup.md

```diff
@@ -3,149 +3,137 @@ id: create-backup
 title: How to backup and restore data
 sidebar_label: Backup and restore data
 ---
+import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
 
 While running, Memgraph generates various files in its [data
 directory](/docs/memgraph/reference-guide/backup), including the **durability
-files**: snapshots and WALs that contain Memgraph's data in a recoverable format
-and are located in the `wal` and `snapshots` folders in the data directory. On
-startup, Memgraph searches for previously saved durability files and uses them to
-recreate the most recent DB state.
+files**, that is,  snapshots and WALs that contain Memgraph's data in a
+recoverable format and are located in the `wal` and `snapshots` folders in the
+data directory. On startup, Memgraph searches for previously saved durability
```

Comment:
i know, but... 
1. this is a how to so this is irrelevant to this how to
2. i don't want to list them ALL because then we will have two places to update if something changes... MVP